cli/replicaset/cartridge.go

```diff
@@ -36,6 +42,13 @@ type CartridgeInstance struct {
 	evaler connector.Evaler
 }
 
+// cartridgeInstCtx describes a cartridge instance.
+type cartridgeInstCtx struct {
+	Failover       Failover
+	ReplicasetUUID string
+	InstanceUUID   string
+}
```

Comment:
I have no idea why we need `Ctx` here.
```suggestion
// cartridgeInstance describes a cartridge instance.
type cartridgeInstance struct {
	Failover       Failover
	ReplicasetUUID string
	InstanceUUID   string
}
```

Reply:
Applied.